### PR TITLE
Rollout12 :ship: 

### PR DESCRIPTION
**Pull request by ivy-rew**
Migrates projects of the axonivy-market org into the new release/12.0 world.

These migrations come with the favour of:
- applying changes to repos consistently
- clearly separating manual commits from manually done work
- giving you a lot of speed

### CODE REVIEW

**Comment by ivy-rew** [comment]
Hey @quanpham-axonivy :wave: 

I saw you filing Release12 migration PR on some projects, this repo has the power to almost fully automate this for you.
Giving you rest from tedious repetitive work and allowing you to keep to be watchful on real changes that may become relevant.
I think it would be best if you continue further migrations throughout the axonivy-market using this script.

If you need help to actually run it; don't hesitate to ask for help.

**Comment by ivy-rew** [comment]
Some examples where this was used:

https://github.com/axonivy-market/graphql-demo/pull/20
https://github.com/axonivy-market/currency-converter-connector/pull/29
https://github.com/axonivy-market/a-trust-connector/pull/49

File: .github/workflows/raise-products.yml [hunk 1]
```diff
@@ -41,10 +41,12 @@ jobs:
       - name: Raise Products
         run: |
           export engineUrl=${{ inputs.engineUrl }}
-          export IVY_JAVA_HOME=$JAVA_HOME_17_X64
+          export IVY_JAVA_HOME=$JAVA_HOME_21_X64
           echo "workDir=$(pwd)" >> $GITHUB_ENV
           ./raise-all-market-products.sh ${{ inputs.version }} ${{ inputs.product }}
         shell: bash
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (ivy-rew) [inline .github/workflows/raise-products.yml:49]: this workflow doesn't really run by now ... as we do not have a single technical octopus (market-org) user that is allowed to file PRs on all repos. I'd vote to introduce one somewhen ...
but for now the pipeline must be run manually:
you may check it out on a simply run `./raise-all-market-products.sh 12.0.0-SNAPSHOT deepl-connector`

**Comment by quanpham-axonivy** [comment]
Thanks a lot @ivy-rew, this is greattt :fire:

**Pull request merged**